PYME/IO/dataExporter.py

```diff
@@ -73,13 +73,15 @@ def Export(self, data, outFile, xslice, yslice, zslice, metadata=None, events =
         pass
 
     def _prepare(self, data, xslice, yslice, zslice, tslice):
-        from PYME.IO.dataWrap import Wrap
+        from PYME.IO import dataWrap
         from PYME.IO.DataSources import BaseDataSource, CropDataSource
-        data = Wrap(data) #make sure we can index along a colour dimension
+        data = dataWrap.Wrap(data) #make sure we can index along a colour dimension
     
         if data.ndim != 5:
-            # promote to xyztc
+            # promote to xyztc and force to array datasource
```

Comment:
Note: data source is not forced to an array datasource - the `XYZTCWrapper` is just a pass-through wrapper which changes the indexing - there is no copy  or array coercion.